_data/sites.json

```
@@ -9135,6 +9154,16 @@
         ]
     },
 
+    {
+        "name": "Surfshark",
+        "url": "https://surfshark.com/privacy-policy",
+        "difficulty": "hard",
+        "notes": "\"If you would like to edit your information or permanently delete your account, you can email us at support@surfshark.com.\"",
```

```suggestion
        "notes": "Send a mail to them requesting for account deletion",
        "email": "support@surfshark.com",
```
We have the `email` field which will automatically create a button to invoke the user's preferred mail client, so I think this can be enhanced as suggested.

---

oh I didn't know thats okay.